src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background: #fff;
+  /* box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.05); */
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
There is no need margins, if you center content in class header. You can just remove it